package.json

```diff
@@ -150,6 +150,9 @@
     "vscode-nls": "^2.0.2",
     "zone.js": "^0.6.26"
   },
+  "resolutions": {
+    "minimatch": "^3.0.2"
```

Comment:
Does the new version of remap-instanbul still not fix this? Or is this for some other package?

But in general it looks like this is again only for these instanbul dependencies, right? Are there any runtime packages using this?

Reply:
it's for another package - `gulp-istanbul-report`, and yes it's only for istanbul dependencies